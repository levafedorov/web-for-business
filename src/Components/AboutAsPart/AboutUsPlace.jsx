import React from 'react';
import  AboutUsArticle from './AboutUsArticle';
import  Adventages from "./Adventages"


export default function AboutUsPlace(props){
     


  

    return <section className="about-us-place" >
       <AboutUsArticle />  
       <Adventages />       
    </section>
}