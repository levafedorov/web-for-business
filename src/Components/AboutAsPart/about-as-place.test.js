import React from 'react';
import {shallow} from 'enzyme';
import AboutUsPlace from './AboutUsPlace';



describe("AboutUsPlace component", () => {
    test("Should AboutUsPlace component render", () => {
        const component = shallow(<AboutUsPlace/>);
        expect(component).toMatchSnapshot();
    });
});