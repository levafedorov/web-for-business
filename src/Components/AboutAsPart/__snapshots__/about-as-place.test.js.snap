// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUsPlace component Should AboutUsPlace component render 1`] = `
<section
  className="about-us-place"
>
  <AboutUsArticle />
  <Adventages />
</section>
`;
